src/i18n/index.ts

```diff
@@ -0,0 +1,47 @@
+import i18n from 'i18next'
+import { initReactI18next } from 'react-i18next'
+import LanguageDetector from 'i18next-browser-languagedetector'
+
+import translation_en from './translation_en.json'
+import translation_fr from './translation_fr.json'
+
+i18n
+  .use(LanguageDetector)
+  .use(initReactI18next)
+  .init({
+    debug: true,
+    detection: {
+      order: ['querystring', 'localStorage', 'navigator', 'htmlTag'],
+      lookupQuerystring: 'lng',
+      lookupLocalStorage: 'i18nextLng',
+      caches: ['localStorage'],
+      htmlTag: document.documentElement
+    },
+    resources: {},
+    fallbackLng: 'en',
+    fallbackNS: 'translation',
+    interpolation: {
+      escapeValue: true,
+      // eslint-disable-next-line @typescript-eslint/typedef
+      format: (value, format): string => {
+        if (!value) {
+          return ''
+        }
+        if (format === 'uppercase') {
+          return value.toUpperCase()
+        }
+        if (format === 'lowercase') {
+          return value.toLowerCase()
+        }
+        return value
+      }
+    },
+    react: {
+      useSuspense: true
+    }
+  })
+
+i18n.addResourceBundle('en', 'translation', translation_en, true)
```

Comment:
In my opinion, should not be the responsibility the i18n service to register translatations (according to my understanding of the sentence: "let each organism declaring its own translation files in json format." of the spec #181).

Reply:
@ccamel 
I'll make the changes 👌

Reply:
@ccamel 
I removed resources management from i18n instance initialization and created some utils methods that can register both default resources and any other resource.
These methods should be described in a Storybook as they will be used both by organisms (for registering its own translations files) and by client app (by deciding, or not, to register default translations and any other resource).
Tell me what do you think ?